include/seqan3/io/stream/parse_condition.hpp

```diff
@@ -558,9 +168,9 @@ constexpr detail::parse_condition_combiner<is_in_interval<'\0', static_cast<char
  * seqan3::is_print(' ');  // returns true.
  * ```
  */
-constexpr is_in_interval<' ', '~'> is_print;
+inline auto constexpr is_print = is_in_interval<' ', '~'> ;
```

Comment:
I would call it `is_printable` .. `is_print` is not very expressive

Reply:
we are using the STL names, as written in the documentation...